sonic_data_client/mixed_db_client.go

```diff
@@ -89,13 +99,49 @@ func getMixedDbClient(zmqAddress string) (MixedDbClient) {
 		} else {
 			client.zmqClient = nil
 		}
-		
+
 		mixedDbClientMap[zmqAddress] = client
 	}
 
 	return client
 }
 
+// This function get target present in GNMI Request and
+// returns: Is Db valid (bool)
+func IsTargetDbByDBKey(dbName string, dbkey swsscommon.SonicDBKey) bool {
+	// In multiple namespaces mode, check namespace
+	// In multiple devices mode, check device
```

Comment:
>multiple devices [](http://example.com/codeflow?start=7&length=16)

difficult to understand 'multiple devices mode'. do you have design doc?

Reply:
Fixed, and HLD PR is https://github.com/sonic-net/SONiC/pull/1625

Reply:
Thanks for improving comment! How about SmartSwitch with a multi-asic NPU? will your design still work?

Reply:
I have updated, now we can support SmartSwitch with a multi-asic NPU.

Reply:
closed.